data/human/news.txt

```diff
@@ -1799,7 +1799,38 @@ news "tourist pa system"
 		word
 			`"`
 
+news "republic navy advisory system"
+	location
+		government "Republic"
+	name
+		"Republic Navy Advisory System"
+	message
+		word
+			`"`
+		word
+			"Please be advised: a travel advisory remains in effect for Greenrock."
+			"Please notify Republic officials immediately if you have been detained by any ship not granted legal authority to do so by the Republic Navy."
+			"Report all sightings of pirate vessels immediately."
+			"The following message is a test. The following message is a test. The following message is a test: Radiation hazard. Evacuate immediately. Radiation hazard. Evacuate immediately. Radiation hazard. Evacuate immediately."
+		word
+			`"`
 
+news "republic navy advisory system [wartime]"
+	location
+		government "Republic"
+	name
+		"Republic Navy Advisory System"
+	to show
+		not "event: fw armistice"
+		has "chosen sides"
+	message
+		word
+			`"`
+		word
+			"Attention: Travel by civilians to rebel-held systems is strictly prohibited."
+			"Attention: Dangerous combat conditions persist in the galactic South. Avoid all travel to affected systems unless strictly necessary."
```

Comment:
I don't think merchants are any less frequent in the South during the war; the FW doesn't attack them.

Reply:
To me this is [a tension between lore and mechanics](https://en.wikipedia.org/wiki/Ludonarrative_dissonance). You're completely right that merchants don't become less frequent, but what government wouldn't at least attempt to prevent trade with their enemy? Besides, you don't know what the origin of any given merchant is - maybe they're from the appropriate side of the frontlines?

Reply:
I guess, how does this effect merchants?

Reply:
There is no mechanical effect on merchant behaviour. It's pure flavour text building upon other lore already within the game.

If there is a strong preference for having the `news` match the ship frequencies , then I'm okay modifying or removing these lines to suit.

Reply:
It's probably fine, I just wanted to note it.